kos-core/source/kos/core/events/DefaultAsyncEventReplier.java

```diff
@@ -1,8 +1,11 @@
 package kos.core.events;
 
+import io.netty.util.internal.ThrowableUtil;
```

Comment:
I'd remove the reliance on Netty's internal. Unfortunately, even between minor improvements on the version 4 we had big changes on its internal classes.